src/pipx/main.py

```diff
@@ -758,11 +758,20 @@ def get_command_parser() -> Tuple[argparse.ArgumentParser, Dict[str, argparse.Ar
         default=0,
         help=(
             "Give less output. May be used multiple times corresponding to the"
-            " WARNING, ERROR, and CRITICAL logging levels."
+            " ERROR and CRITICAL logging levels. Any count > 2 is ignored."
         ),
     )
 
-    shared_parser.add_argument("--verbose", "-v", action="count", default=0, help=("Give more output."))
+    shared_parser.add_argument(
+        "--verbose",
+        "-v",
+        action="count",
+        default=0,
+        help=(
+            "Give more output. May be used multiple times corresponding to the"
+            " INFO, DEBUG and NOTSET logging levels. Any count > 3 is ignored."
```

Comment:
According to my tests, exceeding the count will also result in the `NOTSET` logging level.

```suggestion
            "Give more output. May be used multiple times corresponding to the"
            " INFO, DEBUG and NOTSET logging levels."
```

Reply:
Ah I see how the wording is misleading. Maybe like this:

Reply:
Sounds good!

Reply:
done